python/cugraph/cugraph/datasets/metadata/email_Eu_core.yaml

```
@@ -0,0 +1,30 @@
+name: email-Eu-core
+file_type: .csv
+description:
+  The network was generated using email data from a large European research institution.
+  We have anonymized information about all incoming and outgoing email between members
+  of the research institution. There is an edge (u, v) in the network if person u
```

We should probably go over these and change/remove various words like "we" accordingly, since that could sound like the cugraph team anonymized the data. 
```suggestion
  The network was generated using anonymized email data from a large European research institution. There is an edge (u, v) in the network if person u
```